game/src/game/components/Controllers/AIController.cpp

```diff
@@ -76,72 +73,100 @@ void AIController::OnUpdate(const Timestep& delta_time)
     {
         handling_multiplier_ = kHandlingMultiplierReset;
     }
+}
+
+void AIController::OnUpdate(const Timestep& delta_time)
+{
+    glm::vec3 current_car_position = transform_->GetPosition();
+    glm::vec3 next_waypoint = path_to_follow_[next_path_index_];
+
+    UpdateCarControls(current_car_position, next_waypoint, delta_time);
+    NextWaypoint(current_car_position, next_waypoint);
+}
+
+void AIController::DrawDebugLine(glm::vec3 from, glm::vec3 to)
+{
+    render_service_->GetDebugDrawList().AddLine(
+        LineVertex(transform_->GetPosition()),
+        LineVertex(glm::vec3(path_to_follow_[next_path_index_].x,
+                             path_to_follow_[next_path_index_].y + 10,
+                             path_to_follow_[next_path_index_].z)));
+}
```

Comment:
Not sure if this is intended, but it looks like this doesn't use the `from` or `to` args at all?

Also, instead of drawing a line from player -> next point, it may be more valuable to draw lines between all of the points to see the entire "target" AI path

Reply:
Oh lollll, i made it for that purpose only to have a universal line making function but forgot to change it while refactoring code.